includes/admin/class-wc-admin-notices.php

```diff
@@ -404,7 +404,7 @@ public static function wootenberg_feature_plugin_notice() {
 	 * @since  3.5.1
 	 */
 	protected static function is_ssl() {
-		$shop_page = 0 < wc_get_page_id( 'shop' ) ? get_permalink( wc_get_page_id( 'shop' ) ) : get_home_url();
+		$shop_page = wc_get_page_permalink( 'shop', get_home_url() );
```

Comment:
Thanks for the PR. Outside of my comment above this looks good.

There are 3 coding standards fixes needed https://travis-ci.org/woocommerce/woocommerce/jobs/483386589#L723 for the PR to pass the automated checks. Can you add those as a separate commit?

Reply:
These errors are unrelated to my changes. Should probably be fixed in a separate PR.

Reply:
> These errors are unrelated to my changes.

Yes, but we are asking to have coding standard sniff fixes in files that PRs make changes to included in the PR.

Reply:
Oh ok, wasn't aware of that, we'll do then.